ios/AppLovinMAXNativeAdView.h

```diff
@@ -0,0 +1,42 @@
+#import <React/RCTUIManager.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represents a container view for a native ad.
+ */
+@interface AppLovinMAXNativeAdView : UIView
+
+/**
+ * Ad Unit ID that has an error from loading a native ad
+ */
+@property (nonatomic, strong, readonly, class) NSString *loadErrorAdUnitIdentifier;
+
+/**
+ * Initializes with a bridge.
+ */
+- (instancetype)initWithBridge:(RCTBridge *)bridge;
+
+/**
+ * Loads a native ad, called by Javascript via the View Manager.
```

Comment:
nit but `S` is usually capitalized when people refer to `JavaScript`. It's free to be lexicographically correct.

Reply:
Updated all.